arroyo/processing/processor.py

```diff
@@ -477,6 +492,10 @@ def _run_once(self) -> None:
                             # getting revoked by the broker after reaching the max.poll.interval.ms
                             # Polling a paused consumer should never yield a message.
                             assert self.__consumer.poll(0.1) is None
+
+                            if time.time() - self.__last_pause_ts >= LOGGING_FREQUENCY_SEC:
+                                logger.info("Consumer is paused, polling")
+                                self.__last_pause_ts = time.time()
```

Comment:
Same here, we want to verify that we have been in a paused state without unpausing for long. 
So if we unpause the clock has to be reset.